util/include/json_output.hpp

```diff
@@ -78,4 +80,56 @@ inline std::pair<std::string, std::string> toPair(const std::string &key, const
   return std::make_pair(key, value);
 }
 
+// Build a JSON object.  Supports nested objects.
+class BuildJson {
+ public:
+  inline BuildJson() {}
+
+  // Start a root JSON object.
+  inline void startJson() { str += "\n{\n"; }
+
+  // End the root JSON object.
+  inline void endJson() {
+    if (str.size() >= 2 && str[str.size() - 2] == ',') {
```

Comment:
nit: is it possible to put all expressions in commas? I know that some people don't mind about it, but for some it really reduces readibility of the code.
example to the above:
`if ((str.size() >= 2) && (str[str.size() - 2] == ',')) {`

same comment for the rest of the code

Reply:
Makes sense.